content/teaching-curriculum/chapter1_high_standards

```diff
@@ -0,0 +1,32 @@
+---
+_db_id: 
+content_type: project
+flavours:
+- teaching
+learning_outcomes: null
+prerequisites:
+  hard:
+  - teaching-curriculum/quick-familiarity-1
+  soft: []
+ready: true
+story_points: null
```

Comment:
rather leave out null things. Otherwise I need to make sure the script understands that "null" means. 

There are a few required fields. The ones that aren't required and aren't used can be skipped.